src/helpers/StellarNetwork.ts

```diff
@@ -0,0 +1,5 @@
+export enum StellarNetwork {
+    PUBLIC = 'public',
+    TESTNET = 'testnet',
+    MAINNET = 'mainnet',
```

Comment:
This enum should represent OUR values for the StellarNetwork config, which is just PUBLIC and TESTNET.